sections/tooling/stylelint.js

```diff
@@ -1,6 +1,8 @@
 import md from 'components/md'
 
 const StylelintProcessor = () => md`
+> Beware that due to limitations on what is possible for Stylelint custom processors we cannot support the \`--fix\` option. ⚠️ **If you try to use \`--fix\` you will destroy most of your unsaved work.**
```

Comment:
Let's change `will` to `might`.

Also, super sorry this happened to you, that's a shitty experience for sure. 😕 Maybe there's something we can do from the processor side to detect whether you've used `--fix` and block stylelint from running if that's the case? /cc @emilgoldsmith